docs/user/reference/general/initializing.md

```diff
@@ -174,8 +178,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 }
 ```
 
-The Glean Swift SDK does not support use across multiple processes,
-and must only be initialized on the application's main process.
+The Glean Swift SDK supports use across multiple processes. To configure this you call `initialize` as usual from your main process. From the secondary process you can call `initialize` again with the `dataPath` parameter. This allows you to store data locally in a separate location for the secondary process.
+
+Requirements for a secondary process
+- `initialize` must be called from the main thread
+- The reserved data path `glean_data` cannot be used
```

Comment:
What happens if they try?

Reply:
Fixed in https://github.com/mozilla/glean/pull/2433/commits/4ae32ae928accf9d563d33ec8d07ccdaf39e0443